<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Book_Inventory</class>
 <widget class="QMainWindow" name="Book_Inventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book_Inventory</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{color: rgb(26, 54, 130)}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>951</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWdget {color: rgb(26, 54, 130)}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>430</y>
        <width>148</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {color: rgb(26, 54, 130)}</string>
      </property>
      <property name="text">
       <string>Transaction History</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Exit">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>430</y>
        <width>85</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {color: rgb(26, 54, 130)}</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>430</y>
        <width>138</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {color: rgb(26, 54, 130)}</string>
      </property>
      <property name="text">
       <string>Current Inventory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>430</y>
        <width>116</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {color: rgb(26, 54, 130)}</string>
      </property>
      <property name="text">
       <string>Purchase Book</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>430</y>
        <width>108</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {color: rgb(26, 54, 130)}</string>
      </property>
      <property name="text">
       <string>Remove Book</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>430</y>
        <width>85</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {color: rgb(26, 54, 130)}</string>
      </property>
      <property name="text">
       <string>Add Book</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>651</width>
        <height>211</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>162</number>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Retail Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wholesale Price</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; font-style:italic; color:#274874;&quot;&gt;Welcome to Books Galore Inventory!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>190</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Book Name&lt;span style=&quot; color:#ed0202;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="cancelBox">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>440</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {color: rgb(26, 54, 130); background-color: white}</string>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Book ID&lt;span style=&quot; color:#ed0202;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>290</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Retail Price($)&lt;span style=&quot; color:#ed0202;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>340</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wholesale Price ($)&lt;span style=&quot; color:#ed0202;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="okBox">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>440</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {color: rgb(26, 54, 130); background-color: white}</string>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>390</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ed0202;&quot;&gt;*required fields&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bookNameLineEdit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>180</y>
        <width>251</width>
        <height>33</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="bookIDLineEdit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>230</y>
        <width>251</width>
        <height>33</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="retailPriceLineEdit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>251</width>
        <height>33</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="wholesalePriceLineEdit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>330</y>
        <width>251</width>
        <height>33</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>0</y>
        <width>151</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/small-photo-book_sm.jpg.gif</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="transactionPage">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>541</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="booksAdded">
       <attribute name="title">
        <string>Books Purchased</string>
       </attribute>
       <widget class="QTableWidget" name="transactionsTable">
        <property name="geometry">
         <rect>
          <x>-20</x>
          <y>0</y>
          <width>921</width>
          <height>521</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>225</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Date of Purchase</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Book Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Book ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Price</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>550</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#253a77;&quot;&gt;Total Profit:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>550</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalProfit">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>550</y>
        <width>66</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_3">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
